.travis.yml

```diff
@@ -89,8 +89,11 @@ install:
       cp tmp/usr/share/fonts/truetype/humor-sans/Humor-Sans.ttf ~/.fonts
       cp Felipa-Regular.ttf ~/.fonts
       fc-cache -f -v
+      python setup.py install
```

Comment:
It doesn't work at the moment because the docs needs the rc templates file copied by the setup.py build step. We can fix that but at this stage it's not even clear the the `pip install .` is the right solution. The idea is that it will install into a predictable dir (because it does a flat install) that does not contain the hash of the release and can be configured in codecov or coveralls